source/Halibut/Diagnostics/LogEndpointNormalizer.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace Halibut.Diagnostics
+{
+    public static class LogEndpointNormalizer
```

Comment:
Nit: Could make this a touch sexier and make it an extention method. E.g. `public static Uri Normalize(this Uri endpoint)`